Client/game_sa/CVehicleSA.cpp

```diff
@@ -494,6 +495,22 @@ void CVehicleSA::SetPlaneRotorSpeed(float fSpeed)
     pInterface->m_fPropSpeed = fSpeed;
 }
 
+void CVehicleSA::SetVehicleWheelRotation(float fWheelRot1, float fWheelRot2, float fWheelRot3, float fWheelRot4) noexcept
+{
+    auto pInterface = static_cast<CAutomobileSAInterface*>(GetInterface());
```

Comment:
Maybe better to use one common method for both automobiles and bikes? Because the only difference between them is the number of wheels. There could be a branch inside this function.

Reply:
How are you going to get the vehicle type inside CVehicleSA? or just by counting how many number of wheels rotation being input for SetVehicleWheelRotation?

Reply:
`static_cast<VehicleClass>(vehicleInterface->m_vehicleClass) == VehicleClass::AUTOMOBILE`

Reply:
Alright done!